scripts/qa_crowbarsetup.sh

```diff
@@ -1904,7 +1923,7 @@ function onadmin_allocate()
             echo "Setting last node to SLE12 compute..."
             set_node_role_and_platform ${nodes[1]} "compute" "suse-12.0"
         fi
-        if [ -n "$deployceph" ] && iscloudver 6 ; then
+        if [ -n "$deployceph" ] && iscloudver 6plus ; then
```

Comment:
Inside this block, you have a ` if [[ $cloudsource =~ ^M[1-7]$ ]]` test that is wrong for Cloud 7.